Code/GraphMol/AddHs.cpp

```diff
@@ -319,7 +319,8 @@ void setHydrogenCoords(ROMol *mol, unsigned int hydIdx, unsigned int heavyIdx) {
 }  // end of unnamed namespace
 
 namespace MolOps {
-void addHs(RWMol &mol, bool explicitOnly, bool addCoords) {
+void addHs(RWMol &mol, bool explicitOnly, bool addCoords,
+           const UINT_VECT *onlyOnAtoms) {
```

Comment:
This might be a good use case for HasPropWithValueQuery.  I might even change the overloads:

```
void addHs(RWMol &mol, bool explicitOnly, bool addCoords);
void addHs(RWMol &mol, Query &q, bool explicitOnly, bool addCoords);
```